### DISCUSSION TITLE
[Question]: Web UI Layout style and communication issues

### DISCUSSION THREAD

**Discussion by chengSenLong** [blakeblackshear/frigate#12868] [Ask A Question] [answered]
### What is your question:

- [ ] I deployed frigate using docker-conpose according to the documentation, and then opened the server port 5000 to access the web, as shown in the following figure

<img width="534" alt="e02e461dd72916b4ec10bfe65521425" src="https://github.com/user-attachments/assets/5310df4f-1203-43cc-aa6b-a956241cef7e">
I am a little confused, why is it different from the screenshot in the readme.
<img width="811" alt="1398cd032de77e0753e8f4abb49d84c" src="https://github.com/user-attachments/assets/8608e9ee-0a96-49c2-be28-0d18c41a185a">

- [ ] Because I want to do secondary directional development, I cloned the latest code base on GitHub, then entered the web directory to start, and followed the prompts to set PROXY_HOST=192.168.1.12:5000 npm run dev.But the page cannot be blank normally, and the console reports an error as shown below. The request for the /api/profile path is 404. What is the reason for this?
<img width="536" alt="f00d0a8b8719ed61cd7a941c142c125" src="https://github.com/user-attachments/assets/49107444-f7cd-409b-adf2-9785534970d1">
<img width="540" alt="d6b4c18b76edfd99f9ff07c3535b8a9" src="https://github.com/user-attachments/assets/31dcf23a-acca-402f-9dc5-a4a139e39349">
<img width="534" alt="56693e8a8a9ae05b3937038747ed58f" src="https://github.com/user-attachments/assets/f40643e9-42c2-46cc-8935-f6857f3b8860">

- [ ] Please help me take a look, thank you very much

**Comment by hawkeye217** [comment]
You installed frigate 0.13, not 0.14. What is your docker-compose?

**Reply by chengSenLong** [reply]
I configured it according to the official website documentation
this is
![image](https://github.com/user-attachments/assets/f7606af2-6cf8-4758-a1f4-9940a730aeae)

**Reply by NickM-27** [reply, marked as answer]
The image is likely cached, you need to pull the image again to get the correct update

**Reply by chengSenLong** [reply]
Thanks,I'll try again now
